server/src/resolvers/mutation/user.ts

```diff
@@ -1,4 +1,5 @@
 import { getUserId, Context } from '../../utils'
+import * as bcrypt from 'bcrypt'
```

Comment:
please don't import `*` as it could cause deprecation warnings (experience from libfb-js)
also we don't need the whole bcrypt, only `compare` and `hash`